### DISCUSSION TITLE
OpenNIC on NetFPGA-SUME

### DISCUSSION THREAD

**Discussion by m-liess** [Xilinx/open-nic#12] [Q&A]
Hi everybody, 
at the moment I don't have access to any Xilinx Alveo boards and I was wondering if it was possible to run OpenNIC on the NetFPGA-SUME? Or rather if it is possible to modify the design to work on the SUME... and if this even makes sense? 
I'm thankful for every bit of help!
Best, 
Marco

**Comment by cneely-amd** [comment]
Hi @m-liess, 

Unfortunately, the AMD-Xilinx device families aren't the same and they contain very different hardened IP.  

NetFPGA-SUME contains a much older Virtex-7 series device.  
OpenNIC is written for Alveo boards that contain Virtex UltraScale+ devices, and it requires both the CMAC and QDMA IP.  

In terms of open source project that is like a bridge between these designs, the NetFPGA PLUS is a port of NetFPGA SUME for 2x100Gbps to run on top of OpenNIC shell.  https://netfpga.org/

Best regards,
--Chris
